Feature contributors file

daidoji opened the pull request:
Rough draft of potential CONTRIBUTORS.md file for discussion/review.

m00sey reviewed (commented):
Should this be "contributing.md" I would expect contributors to be a list of people.

m00sey commented on Contributors.md:
```diff
@@ -0,0 +1,76 @@
+[PythonStyleGuide](./ref/PythonStyleGuide.md)  
+
+### Commit Message Guidelines
+There are no hard and fast guidelines (TODO: is this true?) but it is helpful to:
```
I mean we don't have rules, but a descriptive to the point message is ideal.

"Adds selective disclosure mechanism"
"Updates test coverage for coring"

We should discuss if we want to start requiring sign off.

m00sey commented on Contributors.md:
```diff
@@ -0,0 +1,76 @@
+- Message in the appropriate discord channel to let people know about your bug/feature/issue, but remember that maintainers maintain at their own pace and discretion on issues of their choosing.  Its best not to ping them more than once a week.  As with all open source, if its an ultra critical bug/feature for you, cash bounties certainly incentivize people to pay attention and offer to help you  ...
+
+## Code of Conduct and Respect
+Treat people with respect.  (TODO: Should we add a more complex code of conduct?)
```
We have something in discord, we could look at that

m00sey reviewed (changes requested):
Requesting changes so we can either open new issues or get clarity

m00sey commented:
Thanks for the PR!

daidoji commented:
> Should this be "contributing.md" I would expect contributors to be a list of people.

Oh whoops, yeah probably a good call.  I will rename.

daidoji commented:
@m00sey renamed and addressed your points.  Def wanted changes and input from everyone else to provide clarity which is why its in Draft form.  Hopefully, we can discuss it tomorrow.

m00sey commented on Contributing.md:
```diff
@@ -0,0 +1,95 @@
+- Most importantly, the commit message should explain the WHY of all the code changes.  When reviewing, reviewers will be confused if code changes for things you didn't list in the commit aren't there.
+
+### Branching Strategy
+Git flow.  Branch from development, name your feature branches something like `feature-name-of-my-feature` and bugs something like `bug-bug-name` and link the github issue that you (or someone else) def should have created for most non-trivial bugs.  When you have fully implemented or fixed, submit a PR to WebOfTrust/keripy.  If you need feedback it can also be appropriate to submit a **draft** P ...
```
I don't think we need to bring GitFlow into it, I think your description is enough.

daidoji replied:
Good point.  Will take it out.

m00sey reviewed (approved):
not a blocking comment

m00sey commented on Contributing.md:
```diff
@@ -0,0 +1,95 @@
+
+## Introduction
+
+Welcome to keripy's contribution guide! This project aims to develop a reference implementation of the keri standards and protocols developed in WebOfTrust on github. We deeply appreciate the time and effort of contributors like you!
```
s/keri/KERI

pfeairheller commented:
@SmithSamuelM recommendation from 10/19/2023 Dev Meeting to move the licensing section to the top (I suggest under Introduction)
